docs/connectors/io-crd-config/sink-crd-config.md

```diff
@@ -63,30 +63,60 @@ If the node where a Pod is running has enough of a resource available, it is pos
 
 ## Secrets
 
-In Function Mesh, the secret is defined through a secretsMap. To use a secret, a Pod needs to reference the secret. Pods can consume secretsMaps as environment variables in a volume. You can specify the `data` field when creating a configuration file for a secret.
+Function Mesh provides the `SecretsMap` field for Function, Source and Sink in the CRD definition. You can refer to the created secrets under the same namespace and the controller can include those referred secrets. The secrets are provide by `EnvironmentBasedSecretsProvider` which can be used by `context.getSecret()` in Pulsar functions and connectors.
```

Comment:
need a comma after "Source" in the first sentence ("...Function, Source, and Sink..") and a comma before which in the last sentence "... `EnvironmentBasedSecretsProvider`, which..."